client/engine/engine.go

```diff
@@ -170,24 +169,18 @@ func (e *Engine) attemptProgress(objective protocols.Objective) {
 // getOrCreateObjective creates the objective if the supplied message is a proposal. Otherwise, it attempts to get the objective from the store.
```

Comment:
```suggestion
// getOrCreateObjective creates the objective if it doesn't already exist in the store. Otherwise, it returns the objective from the store.
```